SUMMARY.md

```diff
@@ -8,5 +8,6 @@
   - [Meetings](meetings.md)
   - [Roles](roles.md)
   - [Working Open](working-open.md)
+  - [Finance](finance.md)
```

Comment:
```suggestion
  - [Finance 🚧](finance.md)
```

Reply:
Don't think it's necessary to do this in two places.

Reply:
I'd like to know from contents what is under construction, assuming this is a practice we widely adopt. If no objection, this small thing is part of what is letting me be ok with a documentation practice that otherwise feels off-base to me. Please consider.

Reply:
This was added for the coordination practices pr, #29  so mirroring here.